src/app/features/email-client/email-client.service.ts

```diff
@@ -38,7 +42,22 @@ export class EmailClientService {
       return false;
     }
 
-    const mailto = "mailto:" + encodeURIComponent(recipient);
+    const dialogRef = this.dialog.open(EmailTemplateSelectionDialogComponent, {
+      data: entity,
+    });
+    const template = await lastValueFrom(dialogRef.afterClosed());
+
+    if (!template) return false;
+
+    const enc = (v: string) => encodeURIComponent(v);
+    const params: string[] = [];
+    const subject = template.subject?.toString().trim();
+    const body = template.body?.toString();
+
+    if (subject) params.push(`subject=${enc(subject)}`);
+    if (body) params.push(`body=${enc(body)}`);
+
+    const mailto = `mailto:${enc(recipient)}${params.length ? `?${params.join("&")}` : ""}`;
```

Comment:
currently my body text always has an empty line above it in the email app. Can we remove that linebreak?